src/components/EditorWidgets/Markdown/MarkdownControl/Toolbar/Toolbar.js

```diff
@@ -26,26 +27,72 @@ export default class Toolbar extends React.Component {
     };
   }
 
-  render() {
+  getToolbarButtonsList() {
     const {
       onMarkClick,
       onBlockClick,
       onLinkClick,
       selectionHasMark,
       selectionHasBlock,
       selectionHasLink,
+      disabled,
```

Comment:
We could minimize changes to this file by simply adding a `hidden` prop to `ToolbarButton`. What do you think?

Reply:
And if the toolbar button has a hidden property, its render function simply return null ?

Reply:
Yep. I'd usually avoid components managing their own visibility, but in this case it feels worth the trade off.